src/lib/hooks.js

```diff
@@ -108,6 +109,10 @@ const api = {
 
 		createOrUpdateGuest(data);
 	},
+  
+	setLanguage(language) {
```

Comment:
You don't need this method here, you can set the iframe language on the `setLanguage` created on `hooks.js`.